app/core/src/main/java/com/thechance/core/domain/repository/HoneyMartRepository.kt

```diff
@@ -69,7 +69,7 @@ interface HoneyMartRepository {
 
     //region product
     suspend fun createProduct(
-        productName: String, productPrice: Double, productQuantity: String, categoriesId: List<Long>
+        productName: String, productPrice: Double, productQuantity: String, categoriesId: List<Long>,ownerId: Long?
```

Comment:
Same here

Reply:
If i use the marketId, it will conflict with the name of the variable when i use fun insert